ersilia/cli/commands/catalog.py

```diff
@@ -29,7 +29,43 @@ def catalog_cmd():
         default=False,
         help="Show more information than just the EOS identifier",
     )
-    def catalog(local=False, file_name=None, browser=False, more=False):
+    @click.option(
+        "--card",
+        is_flag=True,
+        default=False,
+        help="Use this flag to display metadata for the provided model ID",
```

Comment:
We should rephrase it as: "Use this flag to display model card for a given model ID"

Reply:
this has been addressed @DhanshreeA